### PR TITLE
TSPS-259 Add provenance info to PipelineRunReport in GET PipelineRun result response

### PR DESCRIPTION
**Pull request by mmorgantaylor**
### Description 

Add new required field, `pipelineRunReport`, to the AsyncPipelineRunResult object that's returned when calling GET pipeline run result. 

`pipelineRunReport` contains the following fields regardless of PipelineRun status:
- pipelineName
- pipelineVersion
- wdlMethodVersion

`pipelineRunReport` contains `outputs` if the PipelineRun succeeded. 

### Jira Ticket
https://broadworkbench.atlassian.net/jira/software/c/projects/TSPS/boards/193?selectedIssue=TSPS-259

### CODE REVIEW

File: service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public static String convertToJsonString(Object obj) throws JsonProcessingExcept
   public static final String TEST_WORKSPACE_NAME = "testTerraWorkspaceName";
   public static final String TEST_WORKSPACE_STORAGE_CONTAINER_NAME = "test-bucket-name";
   public static final String TEST_WORKSPACE_GOOGLE_PROJECT = "testGoogleProject";
-  public static final String TEST_WDL_METHOD_VERSION = "0.12.1";
+  public static final String TEST_WDL_METHOD_VERSION = "0.1.12";
```
Comment (jsotobroad) [inline service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java:53]: you didnt like my fake version???

**Review by jsotobroad — APPROVED** [review]
woohoo

File: service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java (hunk 1 above)
Comment (mmorgantaylor) [reply service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java:53]: lol it didn't match the others and there's some cross-talk in the tests

Comment (mmorgantaylor) [reply service/src/test/java/bio/terra/pipelines/testutils/MockMvcUtils.java]: but oops that was actually a signal that something was wrong. reverted and fixed a bug! thanks!!

**Pull request merged**